resdk/resources/collection.py

```diff
@@ -28,11 +28,14 @@ class BaseCollection(BaseResource):
     WRITABLE_FIELDS = ('data', 'description', 'settings', 'descriptor_schema',
```

Comment:
If I remember correctly you should remove the `'data'` from the `WRITABLE_FIELDS` list. Otherwise `BaseResource` expects this is a resource filed and may override the property.